CS_Ed_Week.md

```diff
@@ -11,6 +11,40 @@ A workshop for getting young people interested in computer science
   - Algorithms
   - Debugging
 
+## Part 1: Data Types
+- All programming languages have variables that have a variety of purposes.
+- Variables can be used to store information, edit existing information, or replace information all together.
+- Data Types are the aspect of the things stored in the variable.
+- Sound confusing? Let's explain each data type in further depth, and it'll click.
+### Integers
+- Integers are just what they sound like. 
+- This data type stores numbers, specifically whole numbers as that's what an integer is.
+- They can be edited using standard math operators, such as addition, subtraction, multiplcation, and division.
+  - It's important to note that when doing division on numbers that don't go into each other, the data type switches to a double which will be discussed later.
+- Integers are the most common data type that's used in programming because most programming involves a lot of algorithms and math to do complex things which will also be seen later.
+- It's also important to note that there is a limit on how big and how small the number in an int variable can be.
+- The biggest value that an int can store is 2147483647, and the smallest is -2147483648
+### Floats
+- Floats are integer values that can store bigger and smaller numbers than an integer variable.
+- They do the same things as integer varibles, so all the math and 
+- 
+### Double
+- 
+### Strings
+- Let's say you have the word "apple". The word "apple" is simply a combination of the different letters of the English alphabet.
+- Other examples of words that can be made include "orange" or "program" or "bird"
+- Variables that store this type of information are known as String varibales, they store words.
+- These variables are commonly used to store information that a user inputs to use at later times, but they can also be used to reuse text that shows up in programs.
+- In Python, there are a variety of different methods that can be used to edit these String values, but that's for a later time.
+### Boolean
+- Take a simple light switch.
+- Standard light switches have two states: on or off.
+- We store either "on or off" using a boolean varibale, but instead of using "on or off," we store "true or false".
```

Comment:
one suggestion I have is to add that 1 evaluates to true, and 0 evaluates to false